### PR TITLE
Mention Cider Storm, paren-face in the docs

### PR DESCRIPTION
**Pull request by vemv**

### CODE REVIEW

File: doc/modules/ROOT/pages/debugging/debugger.adoc [hunk 1]
```diff
@@ -6,9 +6,8 @@ http://www.gnu.org/software/emacs/manual/html_node/elisp/Edebug.html[Edebug]. Yo
 
 image::cider_debugger.gif[CIDER Debugger]
 
-WARNING: The debugger currently **does not** support ClojureScript. See
-https://github.com/clojure-emacs/cider/issues/1416[this issue]
```
Comment (bbatsov) [inline doc/modules/ROOT/pages/debugging/debugger.adoc:10]: Why did you remove the link to the ticket?

Comment (vemv) [reply doc/modules/ROOT/pages/debugging/debugger.adoc:10]: My impression was that showing the ticket made sense when there was some sort of intention to implement this, so that one could subscribe to that in-progress discussion.

Now it would be all archeology
